src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2022_07_02_preview/models/_models_py3.py

```diff
@@ -4276,8 +4276,8 @@ class ManagedClusterAzureMonitorProfileKubeStateMetrics(msrest.serialization.Mod
     """
 
     _attribute_map = {
-        'metric_labels_allowlist': {'key': 'metricLabelsAllowlist', 'type': 'str'},
-        'metric_annotations_allow_list': {'key': 'metricAnnotationsAllowList', 'type': 'str'},
+        'metric_labels_allowlist': {'key': 'ksm_metriclabelsallowlist', 'type': 'str'},
```

Comment:
Should not modify the SDK. SDK is generated according to swagger. If you want to rename the attribute, please update swagger

Reply:
yes, this was an accidental copy paste error. Removed